packages/flame_jenny/example/analysis_options.yaml

```diff
@@ -0,0 +1,28 @@
+# This file configures the analyzer, which statically analyzes Dart code to
```

Comment:
This should look like the analysis file of all other examples, i.e. use flame_lint

Reply:
removed in bbf01353a5a89312cd105136eabbf20a2e366a7d

Reply:
Changed lint and ran `melos run analyze`. Changes made in [480ce8e](https://github.com/flame-engine/flame/pull/3086/commits/480ce8e95b56137db2088fbcc27da9966b6f54d1)